pkgs/by-name/in/inngest/package.nix

```
@@ -0,0 +1,46 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+buildGoModule rec {
```

```suggestion
buildGoModule (finalAttrs: {
```
More desired than rec (whenever possible)